### PR TITLE
examples: Stable Diffusion `torch.compile` sample with output image

### PR DESCRIPTION
**Pull request by gs-olive**
# Description

- Tutorial for using torch.compile with Stable Diffusion and Torch-TensorRT
- Demonstration of output images without need to rerun the whole script upon initial compilation of the docs

## Type of change

- Example script

# Checklist:

- [ x ] My code follows the style guidelines of this project (You can use the linters)
- [ x ] I have performed a self-review of my own code
- [ x ] I have commented my code, particularly in hard-to-understand areas and hacks
- [ x ] I have made corresponding changes to the documentation
- [ x ] I have added tests to verify my fix or my feature
- [ x ] New and existing unit tests pass locally with my changes
- [ x ] I have added the relevant labels to my PR in so that relevant reviewers are notified

### CODE REVIEW

**Comment by peri044** [comment]
should be in 2.1.

File: docsrc/index.rst [hunk 1]
```diff
@@ -110,6 +110,8 @@ Tutorials
    tutorials/_rendered_examples/dynamo/torch_compile_resnet_example
    tutorials/_rendered_examples/dynamo/torch_compile_transformers_example
    tutorials/_rendered_examples/dynamo/torch_compile_advanced_usage
+   tutorials/_rendered_examples/dynamo/torch_compile_stable_diffusion
+   tutorials/_rendered_examples/dynamo/torch_compile_stable_diffusion
```
Comment (narendasan) [inline docsrc/index.rst:114]: Does this need to be listed twice?

Comment (gs-olive) [reply docsrc/index.rst]: No, it should not have been - thanks for the catch, it is now fixed.

**Review by narendasan — APPROVED** [review]
LGTM

**Pull request merged**